core/src/test/scala/org/apache/spark/scheduler/TaskSchedulerImplSuite.scala

```diff
@@ -196,6 +196,241 @@ class TaskSchedulerImplSuite extends SparkFunSuite with LocalSparkContext with B
     assert(!failedTaskSet)
   }
 
+  def setupTaskScheduler(clock: ManualClock): TaskSchedulerImpl = {
```

Comment:
nit: `private`

Reply:
there is already a setupScheduler function, perhaps make this name more specific for setupTaskSchedulerForLocalityTests